Python/Product/VSInterpreters/PackageManager/CPythonCondaPackageManagerProvider.cs

```diff
@@ -17,16 +17,43 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.Composition;
-using Microsoft.PythonTools.Infrastructure;
+using System.IO;
```

Comment:
Sort

Reply:
already sorted, that Microsoft line was deleted